vignettes/fileview.Rmd

```diff
@@ -52,8 +52,13 @@ Creating a File View:
 
 ```{r collapse=TRUE}
 view <- EntityViewSchema(name='my first file view',
+                         columns=c(
+                           Column(name="contributor", columnType="STRING"),
+                           Column(name="class", columnType="STRING"),
+                           Column(name="rank", columnType="STRING")),
                          parent=project$properties$id,
-                         scopes=c(project$properties$id))
+                         scopes=c(project$properties$id),
```

Comment:
in this case the concat is unnecessary from a style perspective! either use `scopes=project$properties$id` or make the example use multiple containers.

Reply:
Thanks @kdaily! I will update the style in a separate PR.